openssl/src/asn1.rs

```diff
@@ -705,6 +729,12 @@ impl Asn1Object {
     }
 }
 
+impl Clone for Asn1Object {
+    fn clone(&self) -> Self {
+        unsafe { Asn1Object::from_ptr(cvt_p(ffi::OBJ_dup(self.as_ptr())).unwrap()) }
```

Comment:
Instead of this you can do `clone = OBJ_dup;` in the `foreign_type_and_impl_send_sync` macro.

Reply:
thanks, done